FORMULA

```diff
@@ -0,0 +1,9 @@
+name: apt
+os: Debian, Ubuntu, Raspbian
+os_family: Debian
+version: 0.8.1
+release: 1
+minimum_version: 2017.7
+summary: Packages formula
```

Comment:
I have a PR open on template-formula asking that capitalization be removed here i.e. use "packages" not "Packages".

Reply:
changed